### DISCUSSION TITLE
Outline prohibits embedding in Clickup?

### DISCUSSION THREAD

**Discussion by ReessKennedy** [outline/outline#3684] [General]
We use Clickup for tasks and they have a feature that allows you to create a tab for an embedded, external link to documentation so your docs and tasks are right next to each other. It works well for GoogleDocs and our documentation in Dokuwiki but we are getting blocked when we try to pull in documentation written in outline. (See error below.) Is there a way around this so we can carry on with keeping our tasks and docs connected like this? 

<img width="1269" alt="Screen Shot 2022-06-23 at 2 15 45 PM" src="https://user-images.githubusercontent.com/3970267/175368056-71551005-84d2-4cd1-9d57-65fb1c4b39b6.png">

**Comment by tommoor** [comment]
Outline should allow embedding of shared documents, we don't currently allow embedding of the main application – for one reason it would look strange and be quite confusing. What kind of link are you trying to embed?

**Reply by ReessKennedy** [reply]
@tommoor You mean by "share documents" docs that are publicly shared? I can see that these work when I use Clickup's embed feature but this doesn't give me the ability to edit the page while still inside Clickup and this is what we are seeking to be able to do. 

Clickup has created their own Wiki/Doc tool that is pretty good(!) and nicely integrates with their task app so one can have tasks and docs right next to each other without switching. It's helpful. I figured we could move our docs to Outline and then embed Outline using Clickup's external app embed feature (just like they allow one to embed an GoogleDoc and edit it inside Clickup) but it appears the Origin policy being set by Outline is preventing this?

**Comment by gadgetb0y** [comment]
I'm also having this issue. I'm trying to embed a page in my Home Assistant dashboard, but it doesn't render. (Outline Self-hosted, 0.64.3)

Strangely, this works on the cloud hosted version of Outline.

**Reply by tommoor** [reply]
What kind of link though?

**Reply by gadgetb0y** [reply]
I published a document to the internet. 

![CleanShot 2022-06-28 at 17 32 07](https://user-images.githubusercontent.com/7862224/176315508-505e4d10-d355-44da-aaac-fbce37c2f05c.png)

The document displays fine in its own tab but it will not render in the web page card in Home Assistant (iframe). I had a similar issue with Google Sites because Google includes the HTTP header:

`X-Frame-Options: SAMEORIGIN`

Does Outline include the same?

**Reply by tommoor** [reply]
Not for shared documents in theory
